index.js

```diff
@@ -18,19 +18,28 @@ module.exports.templateTags = [
         displayName: 'Private key file',
         description: 'Path to private key file in PEM format',
         type: 'string'
+      },
+      {
+        displayName: 'GitHub API Root',
+        description: 'The base URL to which to send requests',
+        type: 'string'
       }
     ],
     async run({ context }, ...args) {
       // Destructure id, path from context
-      let { github_app_id: id, github_app_private_key_path: path } = context;
+      let { github_app_id: id, github_app_private_key_path: path, github_api_root: githubApiRoot } = context;
```

Comment:
Could we add a default here?

```suggestion
      let { github_app_id: id, github_app_private_key_path: path, github_api_root: githubApiRoot = 'https://api.github.com' } = context;
```